### PR TITLE
Cross compile streamlet with 2.13

### PR DESCRIPTION
**Pull request by andreaTP**
### What changes were proposed in this pull request?
Cross compile Akka Streamlets with Scala 2.13

### Why are the changes needed?
Let users use Scala 2.13 for Akka Streamlets

### Does this PR introduce any user-facing change?
Yes, Akka Streamlet can be used with the latest Scala

### How was this patch tested?
Running tests and with the sensor-data example (`runLocal`)

### CODE REVIEW

**Review by RayRoestenburg — APPROVED** [review]

**Comment by andreaTP** [comment]
Back to draft since we need this commit: https://github.com/lightbend/cloudflow/pull/876/commits/e82a79073691e1f6f982b8b9afc109135fea63a7

to make this work, but the comment above is very explicit ... 
https://github.com/lightbend/cloudflow/blob/master/core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowBasePlugin.scala#L56-L58

**Comment by andreaTP** [comment]
the failure in CI depends as well on this change:
https://github.com/lightbend/cloudflow/pull/876/commits/e82a79073691e1f6f982b8b9afc109135fea63a7

File: core/project/Dependencies.scala [hunk 1]
```diff
@@ -8,12 +8,14 @@ object Version {
   val AkkaMgmt      = "1.0.8"
   val AkkaGrpc      = "1.0.1"
   val AlpakkaKafka  = "2.0.5"
-  val Scala         = "2.12.11"
-  val ScalaOperator = "2.13.3"
+  // Scala versions are bound to Embedded Kafka:
+  // https://github.com/embeddedkafka/embedded-kafka/issues/202#issuecomment-706951609
```
Comment (RayRoestenburg) [inline core/project/Dependencies.scala:12]: can be removed after the other one.

**Review by RayRoestenburg — APPROVED** [review]
Just fix CloudflowFlinkPlugin re: runner jar.

File: core/sbt-cloudflow/src/sbt-test/sbt-cloudflow/decouple-version/build.sbt [hunk 2]
```diff
@@ -2,7 +2,8 @@ lazy val helloWorld =  (project in file("."))
     .enablePlugins(CloudflowApplicationPlugin, CloudflowAkkaPlugin)
     .settings(
       scalaVersion := "2.12.11",
-      cloudflowVersion := "2.0.10",
+      // TODO: re-enable this after the first Cloudflow release with cross-built runtime
```
Comment (andreaTP) [inline core/sbt-cloudflow/src/sbt-test/sbt-cloudflow/decouple-version/build.sbt:5]: Waiting Monday to merge to raise awareness around this test in "pending" state, cc. @RayRoestenburg

**Pull request merged**